packages/destination-actions/src/destinations/stackadapt-audiences/__tests__/__snapshots__/snapshot.test.ts.snap

```diff
@@ -18,11 +43,36 @@ exports[`Testing snapshot for actions-stackadapt-audiences destination: forwardP
 Object {
   "query": "mutation {
       upsertProfiles(
-        subAdvertiserId: PsAwlRv%,
+        advertiserId: PsAwlRv%,
         externalProvider: \\"segmentio\\",
-        profiles: [{userId:\\"PsAwlRv%\\"}]
+        syncId: \\"6a621eeab9a88ec2f054989bccddeec1d63de12fbc7ac74b2a2a67ab5679033f\\",
+        profiles: \\"[{\\\\\\"userId\\\\\\":\\\\\\"PsAwlRv%\\\\\\"}]\\"
+      ) {
+        userErrors {
+          message
+        }
+      }
+      upsertProfileMapping(
+        input: {
+          advertiserId: PsAwlRv%,
+          mappingSchema: \\"[{\\\\\\"incoming_key\\\\\\":\\\\\\"userId\\\\\\",\\\\\\"destination_key\\\\\\":\\\\\\"userId\\\\\\",\\\\\\"data_type\\\\\\":\\\\\\"string\\\\\\",\\\\\\"is_pii\\\\\\":false}]\\",
```

Comment:
Can we use `externalId` for the `userId`'s destination_key?  destination_key is going to be used as the db column name we store for custom properties.